GOVERNANCE-subprojects.md

```diff
@@ -0,0 +1,176 @@
+
+All active Maintainers of each subproject, as defined in the Contributor Ladder, are
+members of that subproject's Maintainer Committee, which governs that subproject.  The
+subproject's Maintainer Committee is responsible for the following subproject governance
+activities:
+
+* Ensuring that the subproject creates and publishes regular releases;
+* Holding regular, subproject-wide discussions on issues and planning;
+* Monthly review of subproject contributors for advancement on the Contributor Ladder;
+* Making final decisions on subproject changes that involve controversial trade-offs;
+* Responding to security compromise reports;
+* Supporting the Code of Conduct within their subproject and referring violations
+  to the Code of Conduct Committee.
+
+Additionally, the subproject's Maintainer Committee will select, by majority vote, one
+representative to the Steering Committee.  This representative need not
+be a member of the subproject's Maintainer Committee, and will be replaced or
+renewed by the committee annually.
+
+Should a member of the Maintainer Committee cease being active in the subproject,
+violate the Code Of Conduct, or need to be removed for some other reason, they
+may be removed by a 2/3 majority vote of the other Council members, or a
+majority vote of the Steering Committee.
+
+## Steering Committee
+
+The overall [TODO:PROJECTNAME] umbrella project is governed by a Steering
+Committee, which is selected as follows:
+
+* One Maintainer representative from each member subproject
+* [TODO:Number] general "Community Representatives"
+
+<!-- depending on your project, you may want to have other kinds of representatives,
+such as an End User rep or a CNCF SIG rep.  Add those above.  Examples:
+* [TODO:Number] general "Maintainer Representatives"
+* [TODO:Number] general "End-User Representatives"
+You will need to add the election rules for these additional roles as well
+-->
+
+### Steering Committee Duties
+
+The Steering Committee is responsible for the following tasks, any of which may
+be delegated to a person or group selected by the committee:
+
+* Reviewing and deciding on new subprojects to add to [TODO:PROJECTNAME]
+* Arbitrating inter-subproject disagreements
+* Selecting the Code of Conduct Committee and ratifying CoC judgements
+* Removing subprojects which have become inactive
+* Acting on escalated requests from security or code quality issues
+* Resolving issues that individual subprojects are unable to resolve
+* Administering project infrastructure, intellectual property, and resources
+* Determining overall direction for advocacy and marketing
+* Issuing statements on behalf of [TODO:PROJECTNAME] and its subprojects
+* Reviewing and approving Contributor Ladder advancement for participants who
+  work on the overall umbrella project
+
+In performance of these duties, the Steering Committee will hold a monthly meeting
+that is open to all Members.  The Committee may hold additional, closed meetings
```

Comment:
```suggestion
that is open to all project Members.  The Committee may hold additional, closed meetings
```
It took me a few reads of this section to infer that we're talking about all project Members (subproject maintainers) here.

Reply:
not just maintainers, but all contributors.  Changing wording.